specification/metrics/effectivecost.md

```diff
@@ -19,24 +19,25 @@ Effective Cost
 
 ## Description
 
-The cost inclusive of amortized upfront fees, amortized recurring fees, and the usage cost of the line item.
+A charge inclusive of the impacts of all reduced rates and discounts, augmented with the amortization of relevant purchases (one-time or recurring) paid to cover future eligible charges.
```

Comment:
Again not a charge. EffectiveCost as a noun is not a charge.

Reply:
Using 'cost' instead of 'charge':
_A **cost** inclusive of the impacts ..._